client/my-sites/site-settings/reading-newsletter-settings/SubscribeModalSetting.tsx

```diff
@@ -25,6 +32,22 @@ export const SubscribeModalSetting = ( {
 }: SubscribeModalSettingProps ) => {
 	const translate = useTranslate();
 
+	// Construct a link to edit the modal
+	const selectedSiteId = useSelector( getSelectedSiteId );
+	const siteEditorUrl = useSelector( ( state: object ) =>
+		getSiteEditorUrl( state, selectedSiteId )
+	);
+	const themeSlug = useSelector( ( state ) =>
+		getSiteOption( state, selectedSiteId, 'theme_slug' )
```

Comment:
I'm getting `null` for this, not sure why. 🤔 

Alternatively we could use one of these:

- https://github.com/search?q=repo%3AAutomattic%2Fwp-calypso%20requestActiveTheme&type=code
- https://github.com/search?q=repo%3AAutomattic%2Fwp-calypso%20getSiteTheme&type=code

Reply:
I am not sure what's going on with this, sometimes it is randomly not `null` and returns the right `theme_slug`. @tyxla would you know what's going on or what should we use here? 🙏

Reply:
Got it sorted with `useActiveThemeQuery` https://github.com/Automattic/wp-calypso/pull/80705/commits/f0c0cb9a2a6f9f17283b5cf26cddf8b7ef186cee